src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -164,16 +160,15 @@ public function editAction(Post $post, Request $request)
 
         return $this->render('admin/blog/edit.html.twig', [
             'post' => $post,
-            'edit_form' => $editForm->createView(),
-            'delete_form' => $deleteForm->createView(),
+            'form' => $form->createView(),
         ]);
     }
 
     /**
      * Deletes a Post entity.
      *
      * @Route("/{id}", name="admin_post_delete")
-     * @Method("DELETE")
+     * @Method("POST")
```

Comment:
@lyrixx, you could use `_method` field in the form instead of changing the required HTTP method to POST. http://symfony.com/doc/current/form/action_method.html